shared/dataflow/codeql/dataflow/DataFlow.qll

```diff
@@ -30,9 +33,26 @@ signature module InputSig {
     ReturnKind getKind();
   }
 
+  /**
+   * A node in the data flow graph that represents an output.
+   */
   class OutNode extends Node;
 
+  /**
+   * A node in the data flow graph representing the value of an argument after
+   * a function call returns. For example, consider the following C++ code:
```

Comment:
```suggestion
   * an operation that might have changed its state. For example, consider the following C++ code:
```
I base this on the equivalent QLDoc in the Go library:
```
   * A node associated with an object after an operation that might have
   * changed its state.
   *
   * This can be either the argument to a callable after the callable returns
   * (which might have mutated the argument), or the qualifier of a field after
   * an update to the field.
   ```

Reply:
Thanks, I've accepted your suggestion and then made some further edits. 👍